content/documentation/getting-started/index.adoc

```diff
@@ -235,18 +235,22 @@ The above is the default when Kiali is installed on Kubernetes - so to access th
 
 Kiali supports several different login options.
 
-*login*: This option allows a user to log in into Kiali using a username and password. This is the default option if using Kubernetes.
+*login*: This option allows a user to login into Kiali using a username and password. This is the default option if using Kubernetes.
```

Comment:
Please change all "login into Kiali" occurrences to "log in to Kiali".

Reply:
It's fixed. Thanks!